openslides/users/views.py

```diff
@@ -162,6 +164,55 @@ def destroy(self, request, *args, **kwargs):
         self.perform_destroy(instance)
         return Response(status=status.HTTP_204_NO_CONTENT)
 
+    def update(self, request, *args, **kwargs):
```

Comment:
This is too much here. It should be done in a more structured way. Proposal: Send out the signal if any group changes. Do not send just the perms one by one but all new perms at once through the signal. Then use inform_changed_data to send out all received objects. Of course this is a little bit more load but I think groups are not changed so often.

For the case of removed perms we have to discuss with @emanuelschuetze: Do we really want that the client suddenly removes stuff from the view someone is heavily reading at the moment? Think about someone reads a motion detail view with a great poem. Now the admin removes the can_see perm. Boom. I don't like this.